meson.build

```diff
@@ -14,7 +14,7 @@ fs = import('fs')
 py = import('python').find_installation(pure: false)
 tempita = files('generate_pxi.py')
 versioneer = files('generate_version.py')
-
+numpy_dep = dependency('numpy', method: 'auto')
```

Comment:
I think need to bump our minimum version to 1.4.0 for this to work (both in the Meson configuration and in CI)